docs/reference/conditions-reference.md

```diff
@@ -305,6 +321,15 @@ blocks:
       when: "change_in('../Gemfile.lock')"
 ```
 
+### When any file changes, except in the docs directory
+
+```yaml
+blocks:
+  - name: Unit tests
+    run:
+      when: "change_in('/', exclude: ['/docs'])"
```

Comment:
The `exclude` needs to be within the map, e.g. `change_in('/', {exclude: ['/docs']})`